example/tests/helpers.test.js

```diff
@@ -53,4 +53,19 @@ describe("Helpers", () => {
         //Assert
         await expect(progressLoader.exists()).resolves.toBeFalsy();
     });
+
+    it("should enter iFrame and get text", async () => {
+        //Arrange
+        const iFrameSelector = "#mce_0_ifr";
+        const textFrameSelector = "#tinymce p";
+        await page.goto("http://the-internet.herokuapp.com/iframe");
+        await page.waitForSelector(iFrameSelector);
```

Comment:
I suggest moving this waiter inside getFrame helper

Reply:
Moved selector waiter from test to getFrame method in Helper class.